data/committees.js

```diff
@@ -162,12 +170,12 @@ export default [
 			// 	link: 'https://design.uclaacm.com/',
 			// },
 			{
```

Comment:
https://discord.gg/hS62MnPm

just got discord link for design - can we add it before instagram? thanks!